gator_grouper/settings.py

```diff
@@ -14,9 +14,34 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 
+def find_oauth_key():  # pylint: disable=R1710
+    """
+    Look for secret_key.py and return the SECRET_KEY entry in it if the file exists.
+    Otherwise, generate a new secret key, save it in secret_key.py, and return the key.
+    """
+    SECRET_KEY_DIR = os.path.dirname(__file__)
+    SECRET_KEY_FILEPATH = os.path.join(SECRET_KEY_DIR, "secret_key.py")
+    sys.path.insert(1, SECRET_KEY_DIR)
+    if os.path.isfile(SECRET_KEY_FILEPATH):
+        # pylint: disable=import-error
+        from secret_key import OAUTH_SECRET_KEY as key
+
+        return key
```

Comment:
You need to have a return (or exception) case when the oauth key cannot be found.

Reply:
Travis does not like the `raise Exception("Couldn't find the oauth key")` as per what @finneyj2  has informed me

Reply:
@Michionlion our subsequent attempts to add an exception case have only led to the pytest cov. error returning